static/sass/_snapcraft_p-code-copyable.scss

```diff
@@ -1,16 +1,4 @@
 @mixin snapcraft-p-code-copyable {
-  // TODO: Remove this if and when https://github.com/canonical-web-and-design/vanilla-framework/issues/2454
-  // is implemented
-  .p-code-copyable.not-cli {
-    &::before {
-      display: none;
-    }
-
-    .p-code-copyable__input {
-      padding-left: 0.5rem;
-    }
-  }
-
   // TODO: Remove this if and when https://github.com/canonical-web-and-design/vanilla-framework/issues/2453
   // is implemented
   textarea.p-code-copyable__input { // sass-lint:disable-line no-qualifying-elements
```

Comment:
I guess this could/should be removed as well? While we haven't implemented support for textarea in code-copyable, code copyable should not be used at all and should be replaced by code snippet (in this case multiline one).

Reply:
It seems that it was replaced already (on publicise page) so I think this can be removed safely. Unless you use textarea as code copyable elsewhere as well.